client-ts/umd-test-page.html

```diff
@@ -0,0 +1,103 @@
+<html>
```

Comment:
This was a test HTML page I added to do some quick testing to make sure the browser scripts "export" themselves properly to global variables. I figured it didn't hurt to keep it in.